src/common/ContractInteractor.ts

```diff
@@ -22,6 +22,7 @@ import relayHubAbi from './interfaces/IRelayHub.json'
 import forwarderAbi from './interfaces/IForwarder.json'
 import smartWalletFactoryAbi from './interfaces/IWalletFactory.json'
 import tokenHandlerAbi from './interfaces/ITokenHandler.json'
+import penalizerAbi from './interfaces/IPenalizer.json'
```

Comment:
i've added the penalizer to the contract interactor in case we need it, even though we aren't using it right now (i did use it for some manual tests)